static/bidder-info/mediayo.yaml

```diff
@@ -0,0 +1,4 @@
+aliasOf: 'teqblaze'
```

Comment:
You'll need to merge your PR branch with master to pull in the teqblaze adapter to get the test suite to pass.